src/styles/SmallMessageInput.scss

```diff
@@ -1,9 +1,14 @@
 .str-chat__small-message-input {
   display: flex;
+  flex-wrap: wrap;
```

Comment:
and here: `align-items: center;`

Reply:
I just took a look at the UI kit to see how it should look like originally and this wouldn't be enough.